substratest/utils.py

```diff
@@ -24,12 +23,6 @@ def create_archive(tmpdir, *files):
     return zip_folder(str(tmpdir))
 
 
-def camel_to_snake(name):
-    """Convert camel case to snake case."""
-    s = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
-    return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s).lower()
-
-
 def replace_dict_keys(d, converter):
```

Comment:
Since this method is now only used to have a dataset_key on the class ObjectiveDataset, maybe we could remove it entirelyn, update the ObjectiveDataset class to have a datamanager_key property and add a dataset_key @property that would return the datamanager_key value. What do you think?

Reply:
Good Idea, can you do the update or do you want me to do it ?

Reply:
On it :)

Reply:
So it looks like the dataset_key property is not used anywhere, so I just renamed it in data_manager_key and deleted the related code. Could you run the full test suite to make sure it's still ok?

Reply:
The test_network module still passes btw, so I'm fairly confident.